snapm/fsdiff/options.py

```
@@ -99,14 +99,31 @@ def from_cmd_args(cls, cmd_args: Namespace) -> "DiffOptions":
         :returns: A new ``DiffOptions`` instance
         :rtype: ``DiffOptions``
         """
+
+        def get_value(
+            args: Namespace, name: str
```

Ruff (0.14.8)

Unused function argument: `args`

(ARG001)